src/components/artwork_filter/index.tsx

```diff
@@ -0,0 +1,124 @@
+import * as React from "react"
+import * as Relay from "react-relay"
+import styled from "styled-components"
+
+import Artworks from "../artwork_grid"
+import Dropdown from "./dropdown"
+import Headline from "./headline"
+import TotalCount from "./total_count"
+
+const PageSize = 10
+
+interface ArtworkFilterProps extends RelayProps, React.HTMLProps<ArtworkFilter> {
+  filter_artworks: any
```

Comment:
This can simply be removed, because `ArtworkFilterProps` extends `RelayProps` which contains the actual `filter_artworks` typings.